src/Commands/stubs/observer.stub

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace DummyNamespace;
+
+use NamespacedDummyModel;
+
+class DummyClass
```

Comment:
Do you need this?

Reply:
yes. It's in Observer make command. you could check it